backend/corpora/common/utils/exceptions.py

```diff
@@ -25,54 +21,3 @@ class InvalidProcessingStateException(CorporaException):
 
 class NonExistentDatasetException(CorporaException):
     pass
-
-
-class ColorFormatException(ProblemException):
```

Comment:
Moved HTTP-specific exceptions to `backend.corpora.common.utils.http_exceptions`.